src/test/java/com/ccentral4j/CCentralTest.java

```diff
@@ -98,6 +103,16 @@ public void pullConfigLate() throws EtcdAuthenticationException, TimeoutExceptio
     verify(client).fetchConfig();
   }
 
+  /** Increment with groups */
+  @Test
+  public void incGroups() throws Exception {
+    cCentral.incrementInstanceCounter("key", "group1", "group2");
```

Comment:
should we also add a test to make sure the old-behaviour would still work as expected e.g. calling with
```
cCentral.incrementInstanceCounter("key", "value");
```

Reply:
Additional test added